javalib/src/main/scala/java/lang/process/UnixProcess.scala

```diff
@@ -228,12 +223,30 @@ object UnixProcess {
               unistd.execve(c"/bin/sh", newArgv, envp)
             }
           }
-          // The spec of vfork requires calling _exit if the child process fails to execve.
+          /* Older, definitely FreeBSD requirement. No such written
+           * requirement with fork(), but it is probably good idea anyway.
+           *   // The spec of vfork requires calling _exit if the child process
+           *   // fails to execve.
+           */
```

Comment:
```suggestion
          /* Older, definitely FreeBSD requirement. No such written
           * requirement with fork(), but it is probably good idea anyway.
           *
           * The spec of vfork requires calling _exit if the child process
           * fails to execve.
           */
```

Reply:
Eric, 

Than you for the timely & helpful review, especially on a hot summer afternoon.

> Does this allow all the process tests to work now?

Yes, I believe.  CI passed, with the exception of apparently unrelated
(new) multiarch failures.  It works on my machines.  It should
run on your machines. If not, let me know.

Actually `ProcessTest` was the canary, doing its job.
I fixed both underlying defects and also made `ProcessTest`
more robust to a timing assumption it was making in two places.

The initial wait could be improved by the next generation.
 I wanted to get this off the  shipping dock after having my
mind torn apart for days tracking down the defects.

Reply:
re: initial whitespace blocking in comments.  

Based on your suggestion, there may be an evolution available here.
What I was trying to achieve is to distinguish the old and the new
parts of the comment.   I could not use font (italics) and I could
not use the color of the type, so I used indentation.

I could use something like
```
/* Current comment:
 *         Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
 *         Donec tristique varius condimentum. Phasellus auctor ut lacus 
 *        eget sodales.
 *
 * Now obsolete comment:
 *        Suspendisse sed odio nec libero maximus efficitur ac ut felis. 
 *       Aliquam erat volutpat. Morbi malesuada est sit amet venenatis viverra
  */
```
I can move the "Lorem" under the "Current" and the "Suspendisse" under "Now"
or use 1 space indentation under "Lorem", etc.   

I think the old comment have some value for someone trying to follow the
series of changes or understand the changes.    

You are a fresh set of eyes. Should I just delete the old comments
and let folks use "git history"?   Anyone in this code in the future
is in a world filled with regret, so perhaps obsolete comments
will not bring much of either edification or solace.

Reply:
As I was rushing to escape this issue, I had a thought.  Perhaps using greater-than (right angle bracket) ">"
as as a quote-block character?  Something like what is done in Discourse and often in email?
Very rushed and inexact:
```
/* This is the way we do things now.
 * it is far better than what was done in the ">" quoted text below.
 *
 *  > This is the way we used to do things.
 *  > it worked for us for many a year.
 * /
```

Reply:
Eric,  I re-worked the comment sections you identified.

I also got to correct the error code given to `_exit()`. 127 is
the convention for "exec failed" and gives  better clue than 1
in case failure ever happens.

Iterative improvement.